postgres/datadog_checks/postgres/postgres.py

```diff
@@ -385,7 +346,7 @@ def _query_scope(self, cursor, scope, instance_tags, is_custom_metrics, relation
         if scope is None:
             return None
 
-        if scope == REPLICATION_METRICS or not self._is_above([9, 0, 0]):
+        if scope == REPLICATION_METRICS or not is_above(self.version, V9):
```

Comment:
I'm fine with both `is_X_X_or_above(self.version)` or `is_above(self.version, CONSTANT)`, I don't think they are that different. But I'm not a big fan of using both in the check implementation. 
Also, from a language perspective, I feel like `is_8_3_or_above` cleary states that is an inclusive inequality and that leaves it unclear if `is_above` is actually inclusive or not.
Feel free to ignore if you want, this is mostly opinion and a small nit.

Reply:
I like that

Reply:
I like that too 👍